### ISSUE TITLE
pdf export

### ISSUE THREAD

**Issue by rwittler** [husonlab/splitstree6#25]
SVG and PNG export of a splitnetwork works well. But a PDF is not created, no error shown either.

**Comment by husonlab** [comment]
Yeah - im having trouble using a library to create PDF output - I plan to replace that by my own implementation of PDF output / I plan to do that very soon Sent from Daniel Huson's phoneOn 26 Jun 2024, at 12:17, rwittler ***@***.***> wrote:﻿
SVG and PNG export of a splitnetwork works well. But a PDF is not created, no error shown either.

**Comment by husonlab** [comment]
Show work better now
